aclk/mqtt.c

```diff
@@ -359,7 +334,7 @@ int _link_event_loop(int timeout)
 {
     if(mqtt_over_websockets)
```

Comment:
This (among) other things needs to be done with the integration of the disconnect callback. We will need to cleanup various pieces of the code here, so it will be in the upcoming-blocked-by-this-PR issue